src/main/java/tasklist/TaskList.java

```diff
@@ -0,0 +1,81 @@
+package tasklist;
+
+import task.Task;
+
+import java.util.ArrayList;
+
+/**
+ * This class contains the task list.
+ */
+public class TaskList {
```

Comment:
I feel that this class does not do much that an `ArrayList<Task>` cannot do, so I was wondering whether this class serves any other purpose that I might have missed?